metacat-client/src/main/java/com/netflix/metacat/client/Client.java

```diff
@@ -299,7 +299,7 @@ public Client build() {
             }
             Preconditions.checkArgument(host != null, "Host cannot be null");
             if (retryer == null) {
-                retryer = new Retryer.Default(TimeUnit.MINUTES.toMillis(30), TimeUnit.MINUTES.toMillis(30), 0);
+                retryer = new Retryer.Default(TimeUnit.MILLISECONDS.toMillis(500), TimeUnit.MINUTES.toMillis(2), 3);
```

Comment:
Perhaps add a comment here describing the policy? Like "retry upto a period of 2mins"